frontend/src/pages/article/ArticleBox/CardList/Card/Card.tsx

```diff
@@ -27,11 +28,16 @@ function Card({ article }: CardProps) {
   const isArticleList = !!projectId;
   const { postArticleClick } = useArticleClick();
   const { label, bgColor } = ARTICLE_CATEGORY_MAP[category];
-
   return (
     <S.CardContainer>
       <Badge bgColor={bgColor}>
-        {projectTitle} <S.ArrowText>&gt;</S.ArrowText> {label}
+        {showProjectTitleInBadge ? (
+          <>
+            {projectTitle} <S.ArrowText>&gt;</S.ArrowText> {label}
+          </>
+        ) : (
+          label
+        )}
```

Comment:
isArticleList라는 아티클 페이지인지 상세 페이지인지 판별하는 boolean 변수가 이미 있는데,  외부에서 showProjectTitleInBadge 주입 받을 필요 없이 변수명을 바꿔서 이걸 활용해도 되지 않을까요?

```suggestion
        {isArticleList ? (
          <>
            {projectTitle} <S.ArrowText>&gt;</S.ArrowText> {label}
          </>
        ) : (
          label
        )}
```